### PR TITLE
Fix #432: Prepare project documentation for onboarding

### LINKED ISSUE #432: Prepare project documentation for onboarding
Prepare project documentation for enrollment-server-onboarding.

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

File: docs/onboarding/Configuration-Properties.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Configuration Properties
+
+The Onboarding Server uses the following public configuration properties:
+
+## Database Configuration
+
+| Property | Default | Note |
```
Comment (banterCZ) [inline docs/onboarding/Configuration-Properties.md:7]: Do we have any agreement about table formatting in markdown? To format or not to format? I would rather format.

File: docs/onboarding/Configuration-Verification-Providers.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Configuration of Verification Providers
+
+This document describes configuration of providers for personal identity document verification and user presence check.
+
+## Document Verification
```
Comment (banterCZ) [inline docs/onboarding/Configuration-Verification-Providers.md:5]: Formatting nitpick, please an empty line after the header.

**Review by banterCZ — APPROVED** [review]
Well done!

File: docs/onboarding/Configuration-Properties.md (hunk 1 above)
Comment (romanstrobl) [reply docs/onboarding/Configuration-Properties.md:7]: I use the same formatting as in PowerAuth server. If we want to change it, let's discuss it.

**Pull request merged**